pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1859,30 +1868,44 @@ ParticipantStatusTrackingInfo.Kit:
       type: string
     shipper:
       type: string
-### StudyPasswordRequirements
-StudyPasswordRequirements:
+
+StudyPasswordPolicy:
   type: object
   required:
     - minLength
-    - isUppercaseLetterRequired
-    - isLowercaseLetterRequired
-    - isSpecialCharacterRequired
-    - isNumberRequired
+    - minCharClasses
+    - charClasses
+    - allowRepeatedChars
   properties:
     minLength:
       type: integer
-    isUppercaseLetterRequired:
-      type: boolean
-    isLowercaseLetterRequired:
-      type: boolean
-    isSpecialCharacterRequired:
-      type: boolean
-    isNumberRequired:
-      type: boolean
-    maxIdenticalConsecutiveCharacters:
+      minimum: 1
+      maximum: 128
+      example: 8
+      description: minimum number of characters in password
+    minCharClasses:
       type: integer
-      default: 2
       minimum: 0
+      example: 3
+      description: must have a character from at least this number of character classes
+    charClasses:
+      type: array
+      items:
+        type: string
+        enum:
+          - UPPER
+          - LOWER
+          - NUMBER
+          - SPECIAL
```

Comment:
Does Auth0 have any limitations on what is allowed for `SPECIAL` characters?

Reply:
I was curious about this too. They list `!@#$%^&*` as examples of allowed chars. I have tried typing in all the symbols on a standard U.S. keyboard and they all seem to work. I couldn't find any definitive Auth0 documentation on it. I guess anything goes...

Reply:
I'd imagine it's limited to ASCII values, but there's one way to find out!

/goes to add a 😄 to his password